e2e/node_config.go

```diff
@@ -687,7 +687,7 @@ func baseTemplateLinux(t *testing.T, location string, k8sVersion string, arch st
 			"--protect-kernel-defaults":           "true",
 			"--read-only-port":                    "0",
 			"--resolv-conf":                       "/run/systemd/resolve/resolv.conf",
-			"--rotate-certificates":               "false",
+			"--rotate-certificates":               "true",
```

Comment:
Is this change because aks-rp always sends "true"? I remember you mentioning it but have forgotten.

Reply:
in prod RP will always set this to true (unless k8s RBAC is disabled), though in the context of abe2e RP isn't involved whatsoever